src/utils.py

```diff
@@ -65,14 +66,21 @@ def waitUntilVisible(
         return WebDriverWait(self.webdriver, timeToWait).until(
             ec.visibility_of_element_located((by, selector))
         )
-
+    
     def waitUntilClickable(
         self, by: str, selector: str, timeToWait: float = 10
     ) -> WebElement:
         return WebDriverWait(self.webdriver, timeToWait).until(
             ec.element_to_be_clickable((by, selector))
         )
 
+    def checkIfTextPresentAfterDelay(
+        self, text: str, timeToWait: float = 10
+    ) -> bool:
+        time.sleep(timeToWait)
```

Comment:
There might be a fancy way to do this using https://selenium-python.readthedocs.io/waits.html, which polls